README.md

```diff
@@ -6,7 +6,106 @@ The Microsoft Graph Communications API allows developers to programmatically int
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/install-azurerm-ps?view=azurermps-6.8.1)
+* [Skype for Business Online Connector module](https://docs.microsoft.com/en-us/office365/enterprise/powershell/manage-skype-for-business-online-with-office-365-powershell)
+* Install .Net Framework 4.7.1. Some samples will not build if you do not install this.
+* You will need Postman, Fiddler, or an equivalent installed to formulate HTTP requests and inspect the responses.  The following tools are widely used in web development, but if you are familiar with another tool, the instructions in this sample should still apply.
+    + [Postman desktop app](https://www.getpostman.com/)
+    + [Telerik Fiddler](http://www.telerik.com/fiddler)
+
+* [ngrok](https://ngrok.com/) for [Local Testing of the bot](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/articles/index.html)
+
+* If you have Visual Studio Enterprise Subscription, you can [Activate office 365] (https://my.visualstudio.com/)
+
+# Samples
+
+The Graph Calling Bot Samples are split into 2
+categories: Remote Media Bot Samples and Local Media Bot Samples.
+
+## Local media samples
+
+Local media samples gives direct access to the inbound and outbound media streams.  
+
+### [AudioVideoPlaybackBot](Samples\V1.0Samples\LocalMediaSamples\AudioVideoPlaybackBot\README.md)
+
+The AudioVideoPlaybackBot demostrates several features of local media scenarios:
+- Plays a movie in multiple resolutions as the main video output feed.
+- Listens to dominant speaker events and subscribes to inbound video feeds of those participants.
+- Allows switching between screen viewing sharer and viewer, and publishes video through the screen sharing socket.
+
+To run the sample, open [Samples\V1.0Samples\LocalMediaSamples\AudioVideoPlaybackBot.sln](Samples\V1.0Samples\LocalMediaSamples\AudioVideoPlaybackBot.sln)
+
+### [HueBot](Samples\V1.0Samples\LocalMediaSamples\HueBot\README.md)
+
+The HueBot demonstrates local media scenarios.
+- Listens to dominant speaker events and changes the hue color of the dominant speaker video.
+
+To run the sample, open [Samples\V1.0Samples\LocalMediaSamples\HueBot.sln](Samples\V1.0Samples\LocalMediaSamples\HueBot.sln)
+
+### [Policy Recording bot sample](Samples\V1.0Samples\LocalMediaSamples\PolicyRecordingBot\README.md)
+This sample demonstrates how a bot can receive media streams for recording. Please note that the sample does not actually record. This logic is left up to the developer.The system will load the bot and join it to appropriate calls and meetings in order for the bot to enforce compliance with the administrative set policy.
+
+To run the sample, open [Samples\V1.0Samples\LocalMediaSamples\PolicyRecordingBot.sln](Samples\V1.0Samples\LocalMediaSamples\PolicyRecordingBot.sln)
+
+## Remote Media Graph Calling Bot Samples
+
```

Comment:
Add a short explanation of what the Remote Media Graph Calling Bot Samples do - similar to the one at the start of the previous section on Local Media Samples.